server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -720,6 +720,7 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 
     private logSessionEndMetrics(closeType: LambdaCloseType) {
         if (this.sessionMetric?.isCompleted()) {
+            Lumberjack.info("Session metric already completed. Creating a new one.", getLumberBaseProperties(this.documentId, this.tenantId));
             this.sessionMetric = createSessionMetric(
```

Comment:
Put this line above line 722 and capture isCompleted() so that we can easily query how often this happened?

Reply:
Hey Yang..I am not very sure how useful it will be for get the rate at which this is happening. Also, like you said, if we really want that number we can still get it by writing a slightly more complicated query. I am hoping not to add too many non essential log lines around session ends. Please let me know what you think.

Reply:
Sounds good :)